frontend/src/metabase/components/MetadataInfo/ColumnInfoIcon/ColumnInfoIcon.styled.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from "@emotion/styled";
+import { css } from "@emotion/react";
+import { Icon } from "metabase/ui";
+
+export const PopoverHoverTarget = styled(Icon)<{ hasDescription: boolean }>`
+  padding: 0.7em 0.65em;
+  opacity: 0;
+
+  path {
```

Comment:
We don't target html elements like that. Can this be avoided? Could you set the opacity on while icon?

Reply:
This is related to the above issue. To keep the styles that need to be overridden simple I wanted to avoid doing opacity on the actual element. I can encapsulate the styles however and export them to make this less coupled.